components/common/Card.tsx

```diff
@@ -132,14 +132,14 @@ function	CardWithTabs(props: TCardWithTabs): ReactElement {
 }
 
 function	CardBase(props: TCard): ReactElement {
-	const {children, onClick, padding = 'regular', variant = 'surface', ...rest} = props;
+	const {children, onClick, padding = 'regular', variant = 'surface', className, ...rest} = props;
```

Comment:
This change wasn't really related to the PR, but as we're spreading the remaining props, we will be spreading the `className` once again, so invalidating the already set className value